src/lib/components/Accordion/root.svelte

```diff
@@ -18,32 +17,27 @@
 		(SingleAccordionRootProps | MultipleAccordionRootProps);
 
 	export type AccordionContext = {
-		type: Readable<Type>;
-		value: Writable<AccordionRootProps['value']>;
+		type: Type;
+		value: AccordionRootProps['value'];
 	};
 
-	const { getContext, setContext } = uniqueContext<AccordionContext>();
-	export const getAccordionContext = getContext;
+	const { getContext, setContext } = reactiveContext<AccordionContext>();
+	export const getRootCtx = getContext;
 </script>
 
 <script lang="ts">
 	import type { BaseProps } from '$lib/types';
-	import { derived, writable, type Readable, type Writable } from 'svelte/store';
 
 	type $$Props = AccordionRootProps;
 
 	export let type: $$Props['type'] = 'single';
-	const writableType = writable(type);
-	$: if (type) $writableType = type;
-
 	export let value: $$Props['value'] = null;
-	const writableValue = controllableState(value, (v) => (value = v));
-	$: if (value) $writableValue = value;
 
-	setContext({
-		type: derived(writableType, (v) => v),
-		value: writableValue
+	const contextStore = setContext({
+		type: [type ?? 'single', (v) => (type = v)],
```

Comment:
One small thing. In this case the default value gets defined twice (on line 37 and 33) because `$$Props['type']` can be undefined, despite we default it to `single`. Its a very common pattern, and I'm worried its a error prone pattern. In slider component i was starting to wrap all the properties that have default values with `NonNullable<$$Props['type']>` to avoid it, and then you'd not have to re-default the property, but thats also a bit cumbersome.